packages/wallets/src/features/cfd/constants.tsx

```diff
@@ -17,19 +17,17 @@ import { TPlatforms } from '../../types';
 
 export const MarketTypeDetails = {
     all: {
-        description: i18n.t(
-            'Trade swap-free CFDs on MT5 with synthetics, forex, stocks, stock indices, cryptocurrencies and ETFs'
-        ),
+        description: i18n.t('Swap-free CFDs on selected financial and derived instruments.'),
         icon: <AccountsDmt5SwfIcon height={48} width={48} />,
         title: i18n.t('Swap-Free'),
     },
     financial: {
-        description: 'This account offers CFDs on financial instruments.',
+        description: 'CFDs on financial instruments.',
```

Comment:
dont we need i18n.t ??

Reply:
Yes I think we need it, will add it now, thank you @rupato-deriv good catch 😜 

cc: @nijil-deriv